src/test/java/school/lemon/changerequest/java/io/pr1/task2/NymberSwapEngineTest.java

```diff
@@ -0,0 +1,22 @@
+package school.lemon.changerequest.java.io.pr1.task2;
+
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import java.io.*;
+/**
+ * Created by User on 03.02.2017.
+ */
+public class NymberSwapEngineTest {
+
+    @Test
+    public void test()throws  FileNotFoundException{
```

Comment:
Not sure it's a good test.
1. Each test should verify only one case.
2. You are modifying test data, so each time test may get different results.
3. Please verify file content, not just returned value.